pages/hub/insights/index.tsx

```diff
@@ -84,6 +124,104 @@ const InsightsHub: WithPageLayout = () => {
             return <InsightRow key={`insights_${insight.id}`} user={user} insight={insight} />;
           })
         )}
+
+        {!user ? (
+          <>
+            <div className="w-full px-8">
+              <div className="w-full md:w-2/3">
+                <Title level={4} className="text-2xl">
+                  Insights Hub
+                </Title>
+                <Text className="text-base ">
```

Comment:
This text should flow all the way across the screen

Reply:
I think this looks better. thanks for the review 👍
<img width="1440" alt="image" src="https://github.com/open-sauced/app/assets/62995161/f44da33e-ba55-4ce6-a9dd-229953d0bbd9">